src/Symfony/Component/Serializer/Normalizer/PropertyInfoNormalizer.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
+use Symfony\Component\PropertyInfo\PropertyListExtractorInterface;
+use Symfony\Component\Serializer\Extractor\ObjectPropertyListExtractor;
+use Symfony\Component\Serializer\NameConverter\AdvancedNameConverterInterface;
+
+
+/**
+ * Converts between objects and arrays using the PropertyInfo and PropertyAccess component.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+final class PropertyInfoNormalizer implements NormalizerInterface, DenormalizerInterface, NormalizerAwareInterface, DenormalizerAwareInterface, CacheableSupportsMethodInterface
+{
+    use NormalizerAwareTrait;
+    use DenormalizerAwareTrait;
+
+    private $instantiator;
+
+    private $normalizerExtractor;
+
+    private $denormalizerExtractor;
+
+    private $propertyAccessor;
+
+    private $nameConverter;
+
+    private $objectClassResolver;
+
+    public function __construct($instantiator, ObjectPropertyListExtractor $normalizerExtractor, PropertyListExtractorInterface $denormalizerExtractor, PropertyAccessorInterface $propertyAccessor, AdvancedNameConverterInterface $nameConverter = null, callable $objectClassReolver = null)
```

Comment:
We should allow to pass the standard NameConverterInterface here (and pass the extra argument required for the advanced one).